backend/azure-functions/Models/ISettings.ts

```diff
@@ -0,0 +1,7 @@
+
+export interface ISettings {
```

Comment:
https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#user-content-interface:~:text=Don't%20prefix%20with%20I
+
```suggestion
export type Settings {
```
usually when you are not aim to implement an interface, you should make it a type